### PR TITLE
Refactor RAT/Network Type into one place

### PR DESCRIPTION
**Pull request by f3ndot**
#### Agreements
- [x] **I have reviewed and accepted the [guidelines for contributing](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/development/.github/CONTRIBUTING.md) to this project.**
- [x] **I have reviewed and closely followed the [Style Guide](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/wiki/Style-Guide) for this Android app.**

---

#### Overview

A simple refactor to move the central place for getting RAT/Network Type name into the `Cell` class.

---

#### Classification
- [ ] Bugfix (non-breaking change which fixes an existing issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

...

- [x] None of the above, just a refactor. No functional change (hopefully :wink:)

### CODE REVIEW

File: AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Device.java [hunk 1]
```diff
@@ -316,121 +316,11 @@ public String getNetworkTypeName() {
...
-                break;
-            case TelephonyManager.NETWORK_TYPE_UMTS:
-                mNetType = "UMTS";
-                break;
-            case TelephonyManager.NETWORK_TYPE_UNKNOWN:
-                mNetType = "Unknown";
-                break;
-        }
-
+        // TODO: 2016-02-28 Why do we do an assignment? Getters shouldn't have side effects
```
Comment (SecUpwN) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/utils/Device.java]: Thanks for your work, @f3ndot! Would you please remove this `TODO`? I'm already having a hard time cleaning up our old code. If you have to add descriptions, they should be following `Javadoc` standards. Thanks for doing so. :=)

**Comment by f3ndot** [comment]
Don't merge just yet, I need to ensure no other parts of the code relied on that setter side-effect. At work, so I can't exactly verify right now.

**Comment by SecUpwN** [comment]
>Don't merge just yet, I need to ensure no other parts of the code relied on that setter side-effect. At work, so I can't exactly verify right now.

Fair. Take your time, I love what you do with our code.

**Comment by SecUpwN** [comment]
>At work, so I can't exactly verify right now.

Ready to merge now? :)

**Comment by SecUpwN** [comment]
>Don't merge just yet, I need to ensure no other parts of the code relied on that setter side-effect. At work, so I can't exactly verify right now.

@f3ndot, I'd really like to merge your work. Please rebase your pull request, we've had many changes.

**Comment by f3ndot** [comment]
Sorry @SecUpwN I usually do opensource contribs on weekends and this one got away from me

**Comment by f3ndot** [comment]
Should be good to go now once the build passes. My local Android Studio is being a little silly after the lowercasing of classes and I can't test locally :-1:

**Comment by SecUpwN** [comment]
Approved!

**Comment by larsgrefer** [comment]
>My local Android Studio is being a little silly after the lowercasing of classes

@f3ndot Windows, Linux or Mac?

**Comment by larsgrefer** [comment]
@f3ndot Try to run the follwing commands in bash:

```bash
rm -r AIMSICD/src/
git checkout -- AIMSICD/src/
```

**Comment by f3ndot** [comment]
Mac and it's probably the case insensitivity. I'll do the checkout thing
you recommended

**Pull request merged**